app.py

```diff
@@ -0,0 +1,24 @@
+from flask import Flask, render_template, request
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def index():
+    return "Main page for video game inclusion."
+
+
+@app.route('/<resource>', methods=['GET', 'POST'])
```

Comment:
```
$ curl 127.0.0.1:5000/foo
Pull general information on video games that are being requested.
```

This should instead be
```suggestion
@app.route('/videogames', methods=['GET', 'POST'])
```